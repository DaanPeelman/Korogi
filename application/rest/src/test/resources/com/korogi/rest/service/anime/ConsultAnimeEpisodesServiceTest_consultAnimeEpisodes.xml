<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <ANIME  ID="1"
            ANIME_TYPE="TV"
            NAME_EN="Steins;Gate"
            NAME_RO="Steins;Gate"
            START_AIR="2011-04-06"
            END_AIR="2011-09-14"
            SYNOPSIS="Steins;Gate synopsis here"

            CREATION_DATE="2017-07-31 00:00:00"
            CREATED_BY="john.doe"
            MODIFICATION_DATE="2017-07-31 00:00:00"
            MODIFIED_BY="john.doe"
            VERSION="1" />

    <EPISODES   ID="1"
                ANIME_ID="1"
                NAME="Prologue of the Beginning and Ending"
                SYNOPSIS="Prologue of the Beginning and Ending synopsis here"
                DURATION_IN_MINUTES="24"
                AIR_DATE="2011-04-05"

                CREATION_DATE="2017-07-31 00:00:00"
                CREATED_BY="john.doe"
                MODIFICATION_DATE="2017-07-31 00:00:00"
                MODIFIED_BY="john.doe"
                VERSION="1" />

    <EPISODES   ID="2"
                ANIME_ID="1"
                NAME="Paranoia of Time Leaps"
                SYNOPSIS="Paranoia of Time Leaps synopsis here"
                DURATION_IN_MINUTES="24"
                AIR_DATE="2011-04-12"

                CREATION_DATE="2017-07-31 00:00:00"
                CREATED_BY="john.doe"
                MODIFICATION_DATE="2017-07-31 00:00:00"
                MODIFIED_BY="john.doe"
                VERSION="1" />

    <ANIME  ID="2"
            ANIME_TYPE="SPECIAL"
            NAME_EN="Steins;Gate: Egoistic Poriomania"
            NAME_RO="Steins;Gate: Oukoubakko no Poriomania"
            START_AIR="2012-02-22"
            END_AIR="2012-02-22"
            SYNOPSIS="Steins;Gate: Egoistic Poriomania synopsis here"
            PREQUAL_ID="1"

            CREATION_DATE="2017-12-02 00:00:00"
            CREATED_BY="john.doe"
            MODIFICATION_DATE="2017-12-02 00:00:00"
            MODIFIED_BY="john.doe"
            VERSION="1" />

    <EPISODES   ID="3"
                ANIME_ID="2"
                NAME="Steins;Gate: Egoistic Poriomania"
                SYNOPSIS="Steins;Gate: Egoistic Poriomania synopsis here"
                DURATION_IN_MINUTES="24"
                AIR_DATE="2012-02-22"

                CREATION_DATE="2017-07-31 00:00:00"
                CREATED_BY="john.doe"
                MODIFICATION_DATE="2017-07-31 00:00:00"
                MODIFIED_BY="john.doe"
                VERSION="1" />
</dataset>