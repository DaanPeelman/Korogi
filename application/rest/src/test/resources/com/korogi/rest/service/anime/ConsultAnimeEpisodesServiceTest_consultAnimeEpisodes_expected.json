{
  "links": [],
  "content": [
    {
      "name": "Prologue of the Beginning and Ending",
      "synopsis": "Prologue of the Beginning and Ending synopsis here",
      "durationInMinutes": 24,
      "airDate": "2011-04-05",
      "type": "episode",
      "links": [
        {
          "rel": "self",
          "href": "http://localhost/episodes/1"
        }
      ]
    },
    {
      "name": "Paranoia of Time Leaps",
      "synopsis": "Paranoia of Time Leaps synopsis here",
      "durationInMinutes": 24,
      "airDate": "2011-04-12",
      "type": "episode",
      "links": [
        {
          "rel": "self",
          "href": "http://localhost/episodes/2"
        }
      ]
    }
  ],
  "page": {
    "size": 2,
    "totalElements": 20,
    "totalPages": 10,
    "number": 1
  }
}